eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -1,4 +1,86 @@
-function GetOnboardingFileForMoniker($docRepoLocation, $moniker) { 
+# Download the .whl file into the given destination directory.
+function Get-WhlFile {
+  param(
+      [Parameter(Mandatory=$true)] [string]$Library,
+      [Parameter(Mandatory=$true)] [string]$Version,
+      [Parameter(Mandatory=$true)] [string]$Destination,
+      [Parameter(Mandatory=$false)] [string]$ExtraIndexUrl = ""
+  )
+
+  $LibArg= "$Library==$Version"
+  if ($ExtraIndexUrl) {
+    $ExtraIndexArg = "--extra-index-url=$ExtraIndexUrl"
+    Write-Host "python -m pip download --quiet --only-binary=:all: --dest $Destination --no-deps $ExtraIndexArg $LibArg"
+    python -m pip download --quiet --only-binary=:all: --dest $Destination --no-deps $ExtraIndexArg $LibArg
+  } else {
+    Write-Host "python -m pip download --quiet --only-binary=:all: --dest $Destination --no-deps $LibArg"
+    python -m pip download --quiet --only-binary=:all: --dest $Destination --no-deps $LibArg
+  }
```

Comment:
`$PSNativeCommandArgumentPassing = 'Legacy'` is how we've been dealing with this for now. I'm not aware of another pattern for dealing with this at this time. 

No changes needed here at this point.